libpromises/evalfunction.c

```diff
@@ -22,6 +22,7 @@
   included file COSL.txt.
 */
 
+#include <platform.h>
```

Comment:
This header is no longer needed

Reply:
AFAICT, this should be the first `#include` in all our sources. We have seen problems when it wasn't and I was even considering adding a GH action checking its presence.

Reply:
I'll add it as a separate commit ;)